addons/web/static/src/webclient/settings_form_view/settings/setting.js

```diff
@@ -1,29 +1,67 @@
 /** @odoo-module **/
 
 import { escapeRegExp } from "@web/core/utils/strings";
-
-import { Component, useState, useChildSubEnv } from "@odoo/owl";
+import { HighlightText } from "../highlight_text/highlight_text";
+import { session } from "@web/session";
+import { FormLabelHighlightText } from "../highlight_text/form_label_highlight_text";
+import { Component, useState } from "@odoo/owl";
 
 export class Setting extends Component {
     setup() {
         this.state = useState({
             search: this.env.searchState,
             showAllContainer: this.env.showAllContainer,
         });
-        // Don't search on a header setting
-        if (this.props.type === "header") {
-            useChildSubEnv({ searchState: { value: "" } });
-        }
         this.labels = this.props.labels || [];
+        this.labels.push(this.labelString, this.props.help);
+        if (this.props.fieldName) {
+            this.fieldType = this.props.record.fields[this.props.fieldName].type;
+            if (typeof this.props.fieldInfo.modifiers.readonly === "boolean") {
+                this.labelClassName = "o_form_label";
```

Comment:
props to FormLabelHighlightText to remove o_form_label_readonly

Reply:
Hello :wave: 
Any chance you could do the review in one go (through a self-review :) ) ?
Would prevent those many emails notification :innocent: